src/main/groovy/org/gradlefx/tasks/AirPackage.groovy

```diff
@@ -65,6 +69,32 @@ class AirPackage extends DefaultTask {
         showAntOutput ant.properties[ANT_OUTPUT_PROPERTY]
     }
 
+    def initInputOutputFiles() {
+        if (project.type == FlexType.air) {
+            if (flexConvention.air.keystore) {
+                def keystore = project.file(flexConvention.air.keystore)
+                if (keystore.exists()) {
+                    inputs.files keystore.absolutePath
```

Comment:
I think that, by calling inputs.files multiple times, you're overriding the setting instead of adding something to it. So it probably needs to be set in one go.